packages/analytics-types/src/config/react-native.ts

```diff
@@ -2,6 +2,7 @@ import { TransportType } from '../transport';
 import { BrowserConfig, TrackingOptions } from './browser';
 
 export interface ReactNativeConfig extends BrowserConfig {
+  attribution?: ReactNativeAttributionOptions;
```

Comment:
note: keeps attribution option in react native as browser SDK V1 to avoid introducing breaking changes for react native SDK